shell/platform/embedder/tests/embedder_a11y_unittests.cc

```diff
@@ -35,7 +35,7 @@ TEST_F(EmbedderTest, CannotProvideMultipleSemanticsCallbacks) {
   {
     EmbedderConfigBuilder builder(
         GetEmbedderContext(EmbedderTestContextType::kSoftwareContext));
-    builder.SetSoftwareRendererConfig();
+    builder.SetSurface(SkISize::Make(1, 1));
```

Comment:
Here and below `(1, 1)` was the default parameter value... but we only ever defaulted it for the software renderer, now we're consistent across the board and make people announce what they want so readers don't need to check the method declaration.